backend/core/src/seeder/seeds/listings/listing-coliseum-seed.ts

```diff
@@ -53,7 +53,7 @@ const coliseumListing: ListingSeedType = {
   applicationDropOffAddress: null,
   applicationDropOffAddressOfficeHours: null,
   applicationMailingAddress: null,
-  applicationDueDate: getDate(1),
+  applicationDueDate: new Date(getDate(1).setHours(17, 0, 0, 0)),
```

Comment:
Seeds were setting the due date/time to the current date/time. NBD but updated them to 5pm.